### PR TITLE
Add predict.cr

### PR DESCRIPTION
**Pull request by RX14**

### CODE REVIEW

**Comment by RX14** [comment]
I mean it's pretty sciencey but maybe it should go in the misc category.

File: README.md [hunk 1]
```diff
@@ -300,6 +300,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## Science and Data analysis
  * [chizge](https://github.com/aladagemre/chizge) - A Network (Graph) Analysis library, inspired by [NetworkX](https://github.com/networkx/networkx)
  * [machine](https://github.com/mathieulaporte/machine) - Simple machine learning algorithm
+ * [predict.cr](https://github.com/RX14/predict.cr) - Satellite prediction library using the sgp4 model.
```
Comment (veelenga) [inline README.md]: Please avoid dot at the end for consistency

**Comment by RX14** [comment]
Updated.

**Comment by veelenga** [comment]
Thank you!

**Pull request merged**